src/Avalonia.Controls/TopLevel.cs

```diff
@@ -147,6 +148,18 @@ public Size ClientSize
             protected set { SetAndRaise(ClientSizeProperty, ref _clientSize, value); }
         }
 
+        protected virtual ILayoutManager CreateLayoutManager() => new LayoutManager();
```

Comment:
Could we move this down with the other methods? I don't like methods in the middle of properties!

Reply:
Fixed!